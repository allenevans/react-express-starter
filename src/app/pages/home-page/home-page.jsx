import React from 'react';
import { bindActionCreators } from 'redux';
import { connect } from 'react-redux';
import Layout from '../layout';
import './home-page.scss';

export const HomePage = () => (<Layout className='home-page page'>
  <h1>React Express Starter</h1>
</Layout>);

HomePage.propTypes = {};

/* istanbul ignore next */
const mapStateToProps = () => ({});

/* istanbul ignore next */
const mapDispatchToProps = dispatch => ({
  actions: bindActionCreators({}, dispatch),
});

export default connect(mapStateToProps, mapDispatchToProps)(HomePage);
