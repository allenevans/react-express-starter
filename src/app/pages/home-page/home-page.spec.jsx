import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import { HomePage } from './home-page';

const renderer = new ReactShallowRenderer();

describe('HomePage component', () => {
  it('should render the component', () => {
    const tree = renderer.render(<HomePage />);

    expect(tree).toMatchSnapshot();
  });
});
