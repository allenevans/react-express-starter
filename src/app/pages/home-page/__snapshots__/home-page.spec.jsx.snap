// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render the component 1`] = `
<Connect(Layout)
  className="home-page page"
>
  <h1>
    React Express Starter
  </h1>
</Connect(Layout)>
`;
